.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+---
+name: 🙋 New Pull Request
+about: Do you want to change something on parcel?
+---
+
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting since someone might have pushed the same thing before!
+-->
+## ↪️ Pull Request
+<!---
+Provide a general summary of the pull request here
+Does this address an existing issue?
+-->
+
+## 💻 Examples
+
+<!-- Examples help us understand the requested feature better -->
+
```

Comment:
Probably also a good idea to add a checklist to the end of the PR to make sure they pass all the necessary requirements for a PR, something like this:

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions
- [ ] Included links to related issues/PRs